packages/zapp/console/src/components/Entities/VersionDetails/EntityVersionDetails.tsx

```diff
@@ -0,0 +1,73 @@
+import * as React from 'react';
+import { Typography } from '@material-ui/core';
+import { makeStyles, Theme } from '@material-ui/core/styles';
+import { contentMarginGridUnits } from 'common/layout';
+import { WaitForData } from 'components/common/WaitForData';
+import { useTaskTemplate } from 'components/hooks/useTask';
+import { ResourceIdentifier, Identifier } from 'models/Common/types';
+import { DumpJSON } from 'components/common/DumpJSON';
+import { Row } from '../Row';
+import EnvVarsTable from './EnvVarsTable';
+import t, { patternKey } from '../strings';
+import { entityStrings } from '../constants';
+
+const useStyles = makeStyles((theme: Theme) => ({
+  header: {
+    marginBottom: theme.spacing(1),
+    marginLeft: theme.spacing(contentMarginGridUnits),
+  },
+  table: {
+    marginLeft: theme.spacing(contentMarginGridUnits),
+  },
+  divider: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    marginBottom: theme.spacing(1),
+  },
+}));
+
+export interface EntityExecutionsProps {
+  id: ResourceIdentifier;
+}
+
+/** The tab/page content for viewing a workflow's executions */
+export const EntityVersionDetails: React.FC<EntityExecutionsProps> = ({ id }) => {
+  const styles = useStyles();
+
+  // TODO: need to be generic for supporting other type like workflow, etc.
```

Comment:
nit: Ideally `TODO:` need to have Task item mentioned - and new task to cover this work should be created

Reply:
or should I change to NOTE: ?

Reply:
Maybe change to NOTE: here